src/protocols/websocket.js

```diff
@@ -86,7 +86,8 @@ class WSNode extends RTWrapper {
       };
 
       this.client.onerror = error => {
-        const err = (error instanceof Error) && error || new Error(error);
+        const err = error ?
+          new Error(error.message || error) : new Error('Unexpected error');
```

Comment:
Don't we loose the stacktrace returned by Kuzzle?

Reply:
It will be fixed with https://jira.kaliop.net/browse/KZL-1001, nevermind

Reply:
Well, even if it will be fixed, I think you made a valid point. I prefer to fix this now, even if it is redundant with a future task.

Reply:
@Aschen > fixed